dockers/docker-teamd/start.sh

```diff
@@ -3,6 +3,11 @@
 rm -f /var/run/rsyslogd.pid
 rm -f /var/run/teamd/*
 
+for pc in `ip link show | grep PortChannel | awk -F "[ :]" '{print $3}'`;
+do
+    ip link delete dev $pc
+done
```

Comment:
why do we need to delete it here? teamd will delete them. For warm boot, this should let teammgrd to deal with them.

Reply:
teamd will not delete them if it receives SIGTERM/SIGKILL, the other option is to have teammgrd to listen to SIGTERM/SIGKILL and execute the teamd -k command

Reply:
`teamd#supervisord: teammgrd Failed: Cannot allocate memory`
this is the issue met without removing the deprecated netdevs

Reply:
I assume you will make further change to make it teamd docker warm restart compatible?

Reply:
the error you posted is from teammgrd. what memory does teammgrd try to allocate?

Reply:
please comment and mark as temporary solution, the solution is to fix teamd